nncf/torch/hardware/fused_patterns.py

```diff
@@ -320,3 +320,120 @@ def create_h_sigmoid_act() -> GraphPattern:
     main_pattern.add_pattern_alternative(pattern)
 
     return main_pattern
+
+
+# pylint:disable=too-many-statements
+@PT_HW_FUSED_PATTERNS.register(HWFusedPatternNames.SE_BLOCK)
```

Comment:
As far as I remember, we agreed to move the SE pattern to ignore patterns.

Reply:
Should we do this in a separate PR as this change will affect both OV and Torch backends or should we do this in this one?

Reply:
Done